gunrock/app/rw/rw_app.cu

```diff
@@ -0,0 +1,338 @@
+// ----------------------------------------------------------------------------
+// Gunrock -- Fast and Efficient GPU Graph Library
+// ----------------------------------------------------------------------------
+// This source code is distributed under the terms of LICENSE.TXT
+// in the root directory of this source distribution.
+// ----------------------------------------------------------------------------
+
+/**
+ * @file hello_app.cu
+ *
+ * @brief Simple Gunrock Application
+ */
+
+#include <gunrock/gunrock.h>
+#include <gunrock/util/test_utils.cuh>
+#include <gunrock/graphio/graphio.cuh>
+#include <gunrock/app/app_base.cuh>
+#include <gunrock/app/test_base.cuh>
+
+// <DONE> change includes
+#include <gunrock/app/rw/rw_enactor.cuh>
+#include <gunrock/app/rw/rw_test.cuh>
+// </DONE>
+
+namespace gunrock {
+namespace app {
+// <DONE> change namespace
+namespace rw {
+// </DONE>
+
+
+cudaError_t UseParameters(util::Parameters &parameters)
+{
+    cudaError_t retval = cudaSuccess;
+    GUARD_CU(UseParameters_app(parameters));
+    GUARD_CU(UseParameters_problem(parameters));
+    GUARD_CU(UseParameters_enactor(parameters));
+
+    // <DONE> add app specific parameters, eg:
+    GUARD_CU(parameters.Use<int>(    
+         "walk-length",   
+         util::REQUIRED_ARGUMENT | util::OPTIONAL_PARAMETER,  
+         10,   
+         "vertex id", 
+         __FILE__, __LINE__));
+    // </DONE>
+
```

Comment:
I think `seed` for random numbers should be a parameter here, default could be `time(NULL)`, I mention this later on as well.

Reply:
fixed